### ISSUE TITLE
Add column descriptions for INFORMATION_SCHEMA.PARTITIONS

### ISSUE THREAD

**Issue by mjonss** [pingcap/docs#17533]
## Change Request

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
https://docs.pingcap.com/tidb/stable/information-schema-columns only shows the table and its columns, not what the columns mean.

2. Describe your suggestion or addition.

3. Provide some reference materials (such as documents and websites) if you could.
https://dev.mysql.com/doc/refman/8.0/en/information-schema-partitions-table.html

**Comment by Oreoxmt** [comment]
> [docs.pingcap.com/tidb/stable/information-schema-columns](https://docs.pingcap.com/tidb/stable/information-schema-columns) only shows the table and its columns, not what the columns mean.

@mjonss It seems that the description of columns in the `COLUMNS` table is already documented in  https://docs.pingcap.com/tidb/stable/information-schema-columns. 
https://github.com/pingcap/docs/blob/aa1631b8bbddc8d12ad8c8ad4d288c2397b91c3a/information-schema/information-schema-columns.md?plain=1#L81-L108

Does this issue suggest a need to include similar descriptions for columns in https://docs.pingcap.com/tidb/stable/information-schema-partitions?

**Comment by mjonss** [comment]
> @mjonss It seems that the description of columns in the `COLUMNS` table is already documented in https://docs.pingcap.com/tidb/stable/information-schema-columns.
> ... 
> Does this issue suggest a need to include similar descriptions for columns in https://docs.pingcap.com/tidb/stable/information-schema-partitions?

@Oreoxmt yes, sorry for the confusion and wrong link. I have updated the description.

**Comment by Oreoxmt** [comment]
@lcwangchao Could you please take a look at this issue and create a doc PR to add it?
